Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py

```diff
@@ -104,19 +109,41 @@ def get_incidents_by_page(args, page, fields_to_populate, include_context):
     return parsed_incidents
 
 
-def get_incidents(query, time_field, size, from_date, fields_to_populate, include_context):
-    query_size = min(PAGE_SIZE, size)
-    args = {"query": query, "size": query_size, "sort": time_field}
-    if time_field == "created" and from_date:
-        from_datetime = None
+def get_demisto_datetme_format(date_string):
+    if date_string:
+        date_object = None
+        # try to parse date string
         try:
-            from_datetime = parser.parse(from_date)
+            date_object = parser.parse(date_string)
         except Exception:
             pass
-        if from_datetime is None and from_date.strip().endswith("ago"):
-            from_datetime = parse_relative_time(from_date)
+        # try to parse relative time
+        if date_object is None and date_string.strip().endswith("ago"):
+            date_object = parse_relative_time(date_string)
+
+        if date_object:
+            return date_object.astimezone().isoformat('T')
```

Comment:
perhaps we can add a UT to verify that this is the sting format supported in XSOAR